annotation_main:
  visualize_service_name: visualize_cloud_service
  sensor_service_name: sensor_service
  mesh_service_name: mesh_service
  record_service_name: record_segmentation_service
  nearest_radious: 0.002
  LEAF_SIZE: 0.002
  radious: 0.05
  the_number_of_dataset: 1000
  save_base_file_name: object_bbox
  occlusion_object_radious: 0.038
  save_dir: /home/ericlab/tsuchida/2022_11/annotation/YOLO
  
decide_object_position:
  sensor_name: phoxi_camera
  sensor_angle_max: 0.3
  sensor_angle_min: -0.3
  sensor_distance_min: 0.8
  sensor_distance_max: 1.2
  z_position: 0.003
  box_height: 0.01
  box_name: denso_box
  object_height: 0.15
  radious: 0.05

data_transform_pkg:
  f_scale: 0.5
  cx_scale: 0.99
  cy_scale: 0.9

data_3D_to_2D:
  radious_x: 0.05
  radious_y: 0.05